src/test/scala/group/scala/karazin/circe/literal/extras/IterableEncodeSuite.scala

```diff
@@ -0,0 +1,192 @@
+package group.scala.karazin.circe.literal.extras
+
+import munit._
+import org.scalacheck._
+import org.scalacheck.Prop._
+import io.circe.syntax._
+import io.circe.parser
+import io.circe.{Json, JsonObject, Codec}
+import group.scala.karazin.circe.literal.extras.arbitraries.instances.{given, _}
+
+/** !!! by default, scalacheck generates empty container in `forAll` !!! */
+class IterableEncodeSuite extends munit.ScalaCheckSuite:
+
+  override def scalaCheckTestParameters =
```

Comment:
Why is this parameter overridden? After all, we want to test for different values.

Reply:
Fixed, although tests take much more time to complete now.